nordicsemi/version.py

```diff
@@ -37,4 +37,4 @@
 
 """ Version definition for nrfutil. """
 
-NRFUTIL_VERSION = "3.3.2"
+NRFUTIL_VERSION = "3.4.0"
```

Comment:
@rlubos, you asked me if the version should be bumped as part of the PR, which I confirmed. However, in hindsight, I think it is better to leave out the version change here, and we'll instead deal with setting the correct version when we do the next release. Sorry for the mixup.

Reply:
Ok, I've removed that commit.